smoke-tests/src/test/java/io/opentelemetry/smoketest/TestContainerManager.java

```diff
@@ -12,6 +12,8 @@
 public interface TestContainerManager {
   void startEnvironment();
```

Comment:
This method is no longer used in any test class, right? WDYT about adding the `if (!started)` check to it instead of creating a new `startEnvironmentOnce()` method?

Reply:
There are 2 implementations of `TestContainerManager` i didn't want to change both of them. I could remove `startEnvironment` and `stopEnvironment` from `TestContainerManager` interface and add them as abstract to `AbstractTestContainerManager`.

Reply:
If they're not used externally then moving them into the base class would be a great idea.